### DISCUSSION TITLE
Using the OpenScan Mini for hands

### DISCUSSION THREAD

**Discussion by AdieResearch** [OpenScanEu/OpenScan2#47] [Q&A]
I'm currently doing a research project that involves imaging hands of patients that have [scleroderma-related ulcers](https://www.sruk.co.uk/scleroderma/scleroderma-and-your-body/digital-ulcers/). I've wanted to try to model the area of the hands around the ulcer to help with measurement and potentially volume estimation, however patients with this disease find holding their phones (or any object really) very painful. Even a simple reconstruction using structure from motion, can be very challenging and produce poor results. In addition, the patients cannot easily manoeuvre their hands and keep them still, especially when they have to take images themselves.

I realise the OpenScan Mini is designed for objects that sit on the turntable and move consistently throughout the image capture process, however the scanner design lends itself well to a hand being placed within the main ring, which avoids any unnecessary contact with the hand.

I was hoping I could ask for suggestions for imaging of hands in this way, where the skin cannot have anything touching it. I see from examples of good scans that objects can have chalk powder or similar applied to help with imaging, however I believe this would be unsuitable for these patients.

Any advice would be great, thank you in advance.

**Comment by OpenScanFan** [comment]
Hey Adie,

Hmmm...yeah, as you pointed out, this scanner or most other scanners for that matter would have a difficult time scanning living or moving things.

However, just an idea, but have you considered doing a plaster cast of the patients hand? then you could use any scanner to scan the cast

There has been some development in this area that your talking about, here is an article
https://www.jamesdysonaward.org/en-GB/2016/project/curatio-low-cost-dedicated-3d-hand-scanner/
https://www.core77.com/projects/45809/Curatio-Worlds-First-Dedicated-3D-Hand-Scanner

The creators name is Pieter Smakman

This is his LinkedIn info:
https://www.linkedin.com/in/pietersmakman/?originalSubdomain=nl

Hope this helps mate :-)
